kustomize/README.md

```diff
@@ -0,0 +1,176 @@
+# Use Kustomize to deploy Online Boutique with variants
+
+This page walks you through the deployment options for the [Online Boutique](https://github.com/GoogleCloudPlatform/microservices-demo) sample application on a [Google Kubernetes Engine (GKE)](https://cloud.google.com/kubernetes-engine) cluster using Kustomize. Each deployment variant is designed as a [**Kustomize Component**](https://github.com/kubernetes-sigs/kustomize/blob/master/examples/components.md), so you can enable multiple variants at once.
```

Comment:
Part of your prerequisites include deploying on platforms other than GKE. As such, we should remove specific mention of GKE from the overview paragraph.

```suggestion
This page contains instructions on deploying variants of the [Online Boutique](https://github.com/GoogleCloudPlatform/microservices-demo) sample application using [Kustomize](https://kustomize.io/). Each variant is designed as a [**Kustomize Component**](https://github.com/kubernetes-sigs/kustomize/blob/master/examples/components.md), so multiple variants can be composed together in the deployment.
```